api/src/services/import-services/utils/datetime.ts

```diff
@@ -25,3 +32,45 @@ export const formatTimeString = (time?: string): string | undefined => {
     return shortTime.format('HH:mm:ss');
   }
 };
+
+/**
+ * Checks if two date strings are equal.
+ *
+ * Note: This will attempt to unify the formatting between the dates.
+ * ie: 2024-01-01 === 01-01-2024
+ *
+ * @param {string} _dateA - Date string
+ * @param {string} _dateB - Date string
+ * @returns {string | undefined}
+ */
+export const areDatesEqual = (_dateA: string, _dateB: string): boolean => {
+  const format = 'dd-mm-yyyy';
+  const dateA = dayjs(_dateA, format);
+  const dateB = dayjs(_dateB, format);
+
+  return dateA.isValid() && dateB.isValid() && dateA.format(format) === dateB.format(format);
```

Comment:
dayjs does have an `isSame` function, if that works here. 

https://day.js.org/docs/en/query/is-same

Reply:
Though, the equals might also just do the same thing, if its just comparing them as strings

Reply:
Updated.
Yea I think they work the same but the isSame version is cleaner.